ui/utils/nameMapper.js

```diff
@@ -0,0 +1,30 @@
+export const podNameMapper = (serviceMeshName, podName) => {
+  switch (serviceMeshName) {
+    case "istio":
+      if (podName.includes("istio-galley")) return "Istio Galley";
+      else if (podName.includes("istio-ingressgateway")) return "Istio Ingress Gateway";
```

Comment:
Minor nitpick, you can get rid of most the `else` here. Makes it more readable. :smile: